packages/fmg-simple-adjudicator/contracts/SimpleAdjudicator.sol

```diff
@@ -160,11 +161,49 @@ contract SimpleAdjudicator {
         );
     }
 
-    function withdraw(address participant)
+    function recoverDestination(
```

Comment:
I see below that this is called as `participant == recoverDestination(...)`, which made me wonder whether the name is a good description of what the function does. How would you feel about `recoverSigner` or `recoverParticipant`?